src/assets/JourneyWithGo-ABlogSeries/goFundamentals.md

```diff
@@ -0,0 +1,821 @@
+---
+title: Fundamentals of Go - FARMISS
+authorName: Prarup Gurung (प्रारुप गुरुङ्ग)
+authorAvatar: https://avatars.githubusercontent.com/u/8559121?v=4
+authorLink: https://github.com/grgprarup
+createdAt: Oct 26, 2023
+tags: function, array, rune, method, interface, string, struct
+banner: https://blog.jankaritech.com/src/assets/JourneyWithGo-ABlogSeries/images/control-statements-banner.svg
+seriesTitle: Journey With Go - A Blog Series
+episode: 3
+---
+
+**[गो](https://go.dev "Go in Nepali")** **"Journey With Go - A Blog Series"** about the basics of Go `Go Basic`,
+advanced concepts in Go `Go Beyond Basics`, testing with go `Godog as Test Framework` and many more.
+
+This is the third installment of the series. You are here which means you have already read the first and second blog of
```

Comment:
There might be some people who haven't read the previous parts of the series, therefore I would use a modifier like 'most likely' or similar: 

_You are here which means you have most likely already read the first and second blog of the series._